NGINX-Ingress.md

```diff
@@ -36,6 +36,6 @@ spec:
 
 Note that the 'host' value represents the DNS name of a node in the Kubernetes cluster, which in this case uses the skytap.example network that comes with a skytap environment.
 
-Once this is done, you can open the exposed service by entering the host name on the port the ingress controller is bound to. In the case of a bare-metal installation, this will be visible on the NodePort service representing the Ingress controller.
+Once this is done, you can access the exposed service by connecting to the specified hostname on the port the ingress controller is bound to. In the case of a bare-metal installation, this will be visible on the NodePort service representing the Ingress controller.

```

Comment:
I still don't understand what this wants me to do. What am i using to access this? What does that look like? How am i connecting? How do i access this NodePort? None of this is addressed.

Reply:
I literally just want to know what I'm doing here. Do i put the url below into a web browser? What is hapenning??????